src/WebJobs.Script/Config/FunctionsHostingConfigurations.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class FunctionsHostingConfigurations : IFunctionsHostingConfigurations
```

Comment:
nit: I think it's cleaner to name something singular rather than plural i.e `FunctionsHostingConfiguration`